101-sqlazure-ragr/azuredeploy.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "primaryDBResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the source Azure SQL Database Resource Group."
+      }
+    },
+    "primarySQLServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the name of the source Azure SQL Server"
+      }
+    },
+    "primarySQLDBName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the name of the source Azure SQL Database"
+      }
+    },
+    "copyRegion": {
+      "type": "string",
+      "defaultValue": "East US",
+      "allowedValues": [
+        "Japan East",
+        "East US 2",
+        "Southeast Asia",
+        "South Central US",
+        "West Europe"
+      ],
+      "metadata": {
+        "description": "Location of the secondary Azure SQL Read-access online copy"
+      }
+    },
+    "copyServerName":{
+      "type": "string",
+      "defaultValue":"copySQLServer",
```

Comment:
if this needs to be globally unique the either generate a value (with uniquestring) or remove the default value